crates/ironrdp-fuzzing/src/oracles/mod.rs

```diff
@@ -118,3 +118,17 @@ pub fn rdp6_decode_bitmap_stream_to_rgb24(input: &BitmapInput<'_>) {
         input.height as usize,
     );
 }
+
+pub fn cliprdr_format(input: &[u8]) {
```

Comment:
Did you intend to add a new fuzz target? (Much welcomed.) You may have forgotten to commit it.

Reply:
Huh.... no wonder it passed CI the first time... 🤣 Thanks for spotting this!